test/utils/wabt_engine.cpp

```diff
@@ -82,12 +82,18 @@ WasmEngine::Result WabtEngine::execute(
 
     const auto* e = reinterpret_cast<const wabt::interp::Export*>(func_ref);
 
+    const auto func_sig = m_env.GetFuncSignature(e->index);
+
     wabt::interp::TypedValues typed_args;
-    for (const auto arg : args)
+    for (size_t i = 0; i < args.size(); ++i)
```

Comment:
Can you add `assert(args.size() == func_sig.param_types.size())`?

Reply:
Good suggestion. Added.

Reply:
```
fizzy/execute/ecpairing/multipoint 32703759095 ns   32702482549 ns            1
fizzy-bench: /home/builder/project/test/utils/wabt_engine.cpp:86: virtual WasmEngine::Result fizzy::test::WabtEngine::execute(WasmEngine::FuncRef, const std::vector<uint64_t> &): Assertion `func_sig->param_types.size() == args.size()' failed.
```

If this error is reported for `ecpairing`, then something is wrong. According to ecpairing.wat the function takes 0 inputs.